projects/plugins/protect/src/js/components/stat-card/styles.module.scss

```
@@ -0,0 +1,59 @@
+.wrapper {
+	background-color: var( --jp-white );
+	box-shadow: 0px 4px 40px rgba(0, 0, 0, 0.08);
+	border-radius: calc( var( --jp-border-radius ) * 2 ); // 8px;
+	display: flex;
+
+	&.horizontal {
+		align-items: center;
+		padding: calc( var( --spacing-base ) * 2 ); // 16px;
+		width: 358px;
+
+		.info {
+			display: flex;
+			align-items: center;
+			width: 100%;
+			overflow: hidden;
+		}
+
+		.label {
+			margin: 0 calc( var( --spacing-base ) * 2	); // 16px;
+			flex-grow: 1;
+		}
+
+		.icon {
+			margin-bottom: 0;
+		}
+	}
+	
+	&.square {
+		--square-stat-vertical-spacing: calc( var( --spacing-base ) * 2	); // 16px;
+		--square-stat-horizontal-spacing: calc( var( --spacing-base ) * 3	); // 24px;
+		flex-direction: column;
+		justify-content: space-between;
+		width: fit-content;
+		padding: var( --square-stat-vertical-spacing ) var( --square-stat-horizontal-spacing );
+
+		.label {
+			white-space: nowrap;
+		}
```

```suggestion
		}
		
		.value {
			margin-top: calc( var( --spacing-base ) * 0.5 ); // 4px
		}
```
Add `4px` spacing between "Blocked requests Last 30 days" and the stat number.